### PR TITLE
[12.0][crm_lead_code] Update crm_lead_view.xml

### PR DESCRIPTION
**Pull request by dammaksalma**

### CODE REVIEW

**Comment by dammaksalma** [comment]
Add 'name' field to search view on 'view_crm_case_leads_filter'

**Comment by rafaelbn** [comment]
This PR is the same of https://github.com/OCA/crm/pull/290

**Review by keshrath — COMMENTED** [review]
I'm not sure what you're trying to improve with this PR. There is already a name field in this SerachView and it is extended to search for code as well. 

https://github.com/odoo/odoo/blob/12.0/addons/crm/views/crm_lead_views.xml#L388

**Review by oihane — APPROVED** [review]
LGTM

File: crm_lead_code/views/crm_lead_view.xml [hunk 1]
```diff
@@ -28,6 +28,9 @@
             <field name="model">crm.lead</field>
             <field name="inherit_id" ref="crm.view_crm_case_leads_filter" />
             <field name="arch" type="xml">
+                <xpath expr="//field[@name='user_id']" position="before">
+                    <field name="name"/>
```
Comment (sbejaoui) [inline crm_lead_code/views/crm_lead_view.xml:32]: Field name is already in the search view, why you need to add it twice?

**Review by sbejaoui — CHANGES_REQUESTED** [review]

**Pull request closed without merging**